specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/blob.json

```diff
@@ -0,0 +1,1599 @@
+    },
+    "LeaseContainerResponse": {
+      "properties": {
+        "leaseId": {
+          "type": "string",
+          "description": "Returned unique lease ID that must be included with any request to delete the container, or to renew, change, or release the lease."
+        },
+        "leaseTimeSeconds": {
+          "type": "string",
+          "description": "Approximate time remaining in the lease period, in seconds."
+        }
+      },
+      "description": "Lease Container response schema."
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "description": "An error response from the Storage service."
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "description": "A list of additional details about the error."
+        }
+      },
+      "description": "An error response from the Storage service."
+    }
+  },
+  "parameters": {
+    "ContainerName": {
+      "name": "containerName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the blob container within the specified storage account. Blob container names must be between 3 and 63 characters in length and use numbers, lower-case letters and dash (-) only. Every dash (-) character must be immediately preceded and followed by a letter or number.",
+      "maxLength": 63,
+      "minLength": 3,
+      "x-ms-parameter-location": "method"
+    },
+    "ImmutabilityPolicyName": {
+      "name": "immutabilityPolicyName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the blob container immutabilityPolicy within the specified storage account. ImmutabilityPolicy Name must be 'default'",
+      "enum": [
+        "default"
+      ],
+      "x-ms-parameter-location": "method"
+    },
+    "BlobServicesName": {
+      "name": "BlobServicesName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the blob Service within the specified storage account. Blob Service Name must be 'default'",
+      "enum": [
+        "default"
+      ],
+      "x-ms-parameter-location": "method"
```

Comment:
Could you please apply the changes in this PR to your swagger to keep things consistent?
https://github.com/Azure/azure-rest-api-specs/pull/17027/files